desktop_version/src/main.cpp

```diff
@@ -632,6 +652,8 @@ int main(int argc, char *argv[])
     }
 
     cleanup();
```

Comment:
Any way to call `cleanup()` when the loop ends?

Reply:
I don't think so, but I could be wrong.

Reply:
No big deal to me either way. It's only needed to minimize Valgrind memory leak messages.